AndroidLibV2rayLite/CoreI/Status.go

```diff
@@ -6,6 +6,7 @@ import (
 
 type Status struct {
 	IsRunning       bool
+	IsTRunning      bool
```

Comment:
命名不应该使用“意义不明确的缩写”，这点各大编码规范中都有提及。建议
```suggestion
	IsTun2socksRunning bool
```

Reply:
do not use variable too long.

Reply:
> do not use variable too long.

Maybe "IsT2SRunning" is better?